### PR TITLE
Update 40_WYSIWYG.md

### PR DESCRIPTION
**Pull request by j-jonas**
How to enable TinymceBundle

  
  

## Changes in this pull request  
Resolves #

### <samp>🤖 Generated by Copilot at 0e3d53a</samp>

This pull request improves the documentation for the WYSIWYG editable by adding a prerequisite step and correcting a layout problem.

### <samp>🤖 Generated by Copilot at 0e3d53a</samp>

> _Sing, O Muse, of the skillful scribe who updated the WYSIWYG_
> _`documentation`, to guide the users of the `TinyMce` bundle_
> _That they may craft their texts with ease and grace, and not be troubled_
> _By the formatting of notes, which he corrected with a subtle_

### <samp>🤖 Generated by Copilot at 0e3d53a</samp>

*  Add a new subsection to explain how to enable the TinyMce bundle for WYSIWYG fields ([link](https://github.com/pimcore/pimcore/pull/16177/files?diff=unified&w=0#diff-c601cb407d4defaaf4f0ab272abee11dcc09f7984d5f296e824f86e72ed7e46aR7-R17))
*  Remove a redundant line break in the note about HTML encoding in the sanitizer configuration section ([link](https://github.com/pimcore/pimcore/pull/16177/files?diff=unified&w=0#diff-c601cb407d4defaaf4f0ab272abee11dcc09f7984d5f296e824f86e72ed7e46aL58-R69))

### CODE REVIEW

File: doc/03_Documents/01_Editables/40_WYSIWYG.md [hunk 1]
```diff
@@ -4,6 +4,17 @@
...
 
+## Enable TinyMce
+
+Add the bundle in your `config/bundles.php` file:
+
+```php
+\Pimcore\Bundle\TinymceBundle\PimcoreTinymceBundle::class => ['all' => true],
+```
+
+After this be sure, that PimcoreTinymceBundle is installed. You can check it with: `bin/console pimcore:bundle:list`
```
Comment (mcassier31) [inline doc/03_Documents/01_Editables/40_WYSIWYG.md]:
```suggestion
After this, make sure PimcoreTinymceBundle is installed with `bin/console pimcore:bundle:list`.
```

File: doc/03_Documents/01_Editables/40_WYSIWYG.md [hunk 2]
```diff
@@ -4,6 +4,17 @@
...
+If it is not installed, you can install it with: `bin/console pimcore:bundle:install PimcoreTinymceBundle`
```
Comment (mcassier31) [inline doc/03_Documents/01_Editables/40_WYSIWYG.md]:
```suggestion
If it is not installed, you can install it with `bin/console pimcore:bundle:install PimcoreTinymceBundle`.
```

**Review by mcassier31 — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution! 
For consistency reasons, could you apply the following minor changes, please? :)

**Comment by j-jonas** [comment]
Done

**Pull request merged**